infra.bs

```diff
@@ -985,7 +985,10 @@ literal, we can use plainer language: <var>domain</var> <a for=string>ends with<
 {{Array/sort()}} method on an array of strings. This ordering compares the 16-bit code units in each
 string, producing a highly efficient, consistent, and deterministic sort order. The resulting
 ordering will not match any particular alphabet or lexicographic order, particularly for
-<a>code points</a> represented by a surrogate pair. [[!ECMA-262]]
+<a>code points</a> represented by a surrogate pair. For example, the codepoint U+FF5E
```

Comment:
Optional: you could split this from the note `<p>` and put it in a separate `<p class="example">`